docs/core/run-time-config/garbage-collector.md

```diff
@@ -290,6 +292,39 @@ Example:
 > [!TIP]
 > If you're setting the option in *runtimeconfig.json*, specify a decimal value. If you're setting the option as an environment variable, specify a hexadecimal value. For example, to limit the heap usage to 30%, the values would be 30 for the JSON file and 0x1E or 1E for the environment variable.
 
+### Per-heap limits
```

Comment:
Shall we say `Per-object-heap` limits instead of `Per-heap` limit to avoid confusion?
@Maoni0

Reply:
yes I would say per-object-heap.